src/structures/ThreadChannel.js

```diff
@@ -109,6 +117,9 @@ class ThreadChannel extends Channel {
       if (!this.archiveTimestamp) {
         this.archiveTimestamp = null;
       }
+      if (!this.invitable) {
+        this.invitable = null;
+      }
```

Comment:
Since we're using node v16.6 now, why not change this and all the other ones above to the following
```suggestion
        this.invitable ??= null;
```